### DISCUSSION TITLE
prune-ckpt.py

### DISCUSSION THREAD

**Discussion by SingingTurnip** [JoePenna/Dreambooth-Stable-Diffusion#56] [Q&A]
Hello, 

How can I run the prune-ckpt.py file to shrink down my model file without a full install of this version of Stable Diffusion, is that possible?

I use a colab for DB as I dont have a 3080 lol and I have Automatic1111 for SD itself.

**Comment by nawnie** [comment]
You could always just load this colab after you are done and place the .ckpt in your root folder. This may be the simplest way depending on your coding experience, I don't believe premium gpu is needed for this
